data.sql

```diff
@@ -21,4 +21,31 @@ VALUES
     (4, 'Angemon', '2005-06-12', 1, TRUE, -45),
```

Comment:
- [ ] In your readme, you have a Live Demo link. it is not required as this project doesn't have that. It is also leading to 404 page. I request you to kindly remove that from Readme.